make-up/code/5163-mihaela-milosevska/src/app/recipe-list/recipe-list.component.html

```diff
@@ -1 +1,51 @@
-<p>And the recipe list and everything else comes here</p>
+<div>
+    <button (click)="addRecipe()">Add Recipe</button>
+</div>
+
+<table>
+  <thead>
+    <tr>
+      <th>ID</th>
+      <th>Title</th>
+      <th>Cusine</th>
+      <th>Description</th>
+      <th>Ingredients</th>
+      <th>Instructions</th>
+      <th>Time</th>
+      <th>Servings</th>
+      <th>Actions</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr *ngFor="let Recipe of Recipes">
```

Comment:
The variable in the code is called `recipes` not `Recipes`, and the loop variable used bellow is written both as `Recipe` and `recipe` which is not working,